### ISSUE TITLE
Does this image have the ability to change the game server panel port from 8080 to something else?

### ISSUE THREAD

**Issue by mikedevita** [wine-gameservers/arch-wine-fs22#14]
When I try to change the bind from `8080:8080` to `8088:8080` the panel still tries to redirect to port :8080.

Is there a way to override this functionality?

**Comment by b1ggi** [comment]
use this:
![image](https://github.com/user-attachments/assets/9c6c583e-813d-4e11-bbfe-352e2d67fe52)
and use the link:
http://192.168.x.y:8086/index.html

**Comment by mikedevita** [comment]
Yes, but when I access the web UI on the new port after I login it's trying to redirect me to http://192.168.x.y:8080/index.html

**Comment by Toetje585** [comment]
> Yes, but when I access the web UI on the new port after I login it's trying to redirect me to http://192.168.x.y:8080/index.html

This wil be fixed with a upcoming wine version where we can drop nginx for good.

**Comment by AlexR2106** [comment]
Hello,
I also needed to Change to Port and done a little Workaround:
Added an additional Volume:
- /opt/docker/fs22/nginx:/etc/nginx/sites-enabled

There you can place the fs22 nginx config file and adjust the listenting port according to your needs. Do not forget to adjust the ports section accoring to your configuration.
